en/docs/observability/root-cause-analysis.md

```diff
@@ -0,0 +1,99 @@
+# Root Cause Analysis
+
+Performing root cause analysis is crucial in identifying and rectifying the underlying problem for any anomalies detected in a system. Choreo Observability provides helpful insights out-of-the-box that assists in carrying out a root cause analysis for any service deployed on the Choreo platform.
+
+!!! info
+    For explanation, we are using the sample data available in the product.
+    
+## Get started with the sample service
+
+1. Go to **Services** from the left navigation.
+2. If this is your first service, then enter a name for the service and click **Create**. Skip step 3 and go to step 4.
+       ![Create a new service](/assets/img/observability/first-service-creation.png)
+3. Else, click **+ Create** to create a new service.
+       ![Create a new service](/assets/img/observability/service-creation.png)
+    1. Enter a name for the service and click **Create**.
+4. From the left-hand menu, click **Observe**.
+5. On the right-hand bottom, click the **Sample Service** icon.
```

Comment:
In some places, we refer to the UI elements with the following word `icon` and some places not. Is that intentional?

Reply:
Yes, all other places are labels. This is just an icon at the bottom corner, with no label.